### PR TITLE
Adds Dmitri as an editor, adds link to Go implementation

### PR DESCRIPTION
**Pull request by swcurran**

### CODE REVIEW

File: spec/implementors_guide.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 Proof of concept implementations of `did:tdw` software for [[ref: DID Controllers]] and resolvers can be found here:
 
-- Typescript: [https://github.com/bcgov/trustdidweb-ts](https://github.com/bcgov/trustdidweb-ts)
-- Python: [https://github.com/bcgov/trustdidweb-py](https://github.com/bcgov/trustdidweb-py)
+- [Typescript]
```
Comment (brianorwhatever) [inline spec/implementors_guide.md:7]: I don't know how these square brackets work without the parenthesis after.. Does this link to the header?

File: spec/implementors_guide.md [hunk 2]
```diff
@@ -4,8 +4,9 @@
 
 Proof of concept implementations of `did:tdw` software for [[ref: DID Controllers]] and resolvers can be found here:
```
Comment (brianorwhatever) [inline spec/implementors_guide.md:5]: If those links don't link we shouldn't have "found here"

File: spec/implementors_guide.md [hunk 3]
```diff
@@ -4,8 +4,9 @@
...
 
-- Typescript: [https://github.com/bcgov/trustdidweb-ts](https://github.com/bcgov/trustdidweb-ts)
-- Python: [https://github.com/bcgov/trustdidweb-py](https://github.com/bcgov/trustdidweb-py)
+- [Typescript]
+- [Python]
+- [Go]
 
 Both currently (as of 2024.04.11) support all of the features of the core `did:tdw` including [[ref: Key Pre-Rotation]]. Not yet supported is the the concept of [[ref: witnesses]].
```
Comment (brianorwhatever) [inline spec/implementors_guide.md:11]: Maybe this isn't required anymore? If we want to keep it should probably be update

**Review by brianorwhatever — APPROVED** [review]

File: spec/implementors_guide.md (hunk 2 above)
Comment (swcurran) [reply spec/implementors_guide.md:5]: Discovered that as well — but decided not to fix in this PR. Will be adding another refs and defs PR right after this.

File: spec/implementors_guide.md (hunk 3 above)
Comment (swcurran) [reply spec/implementors_guide.md:11]: Thanks — next PR.

**Comment by swcurran** [comment]
Closing this.  We've moved the repo into DIF, so will recreate this PR over there.

**Pull request closed without merging**